index.html

```diff
@@ -2445,24 +2447,92 @@ <h2>Verification Algorithm</h2>
 As a general rule, verifiers SHOULD strive to minimize the processing of untrusted data.
 This includes minimizing any processing of the protected header, unprotected header, or payload as part of the key discovery procedures.
     </p>
-
     <p>
-When verifying a credential or presentation secured with SD-JWT, the algorithm defined in
-[[SD-JWT]] for
-  Verification of the SD-JWT
-MUST be followed.
+After verification has succeeded, additional validation checks SHOULD be
+performed as described in Section <a href="#validation-algorithm"></a>
     </p>
-
+    <p>
+The outputs for the following algorithms are:
+    </p>
+    <ul>
+      <li>
+<code>status</code>: a boolean indicating the result of verification,
+<code>true</code> for success and <code>false</code> for failure.
+        </li>
+      <li>
+<code>document</code>: a document conforming to the [[VC-DATA-MODEL-2.0]]
+        </li>
+      <li>
+<code>mediaType</code>: <code>vc+ld+json</code>
+        </li>
+    </ul>
+    <section>
+      <h3>Algorithm for verifying a credential or presentation secured with SD-JWT</h3>
+      <p>
+The inputs for this algorithm are:
+      </p>
+      <ul>
+        <li>
+<code>inputMediaType</code>: <code>vc+ld+json+sd-jwt</code>
+          </li>
+        <li>
+<code>inputDocument</code>: the verifiable credential secured with [[SD-JWT]]
+          </li>
+      </ul>
+      <p>
+Upon receipt of the verifiable credential secured with [[SD-JWT]], the holder or
+verifier follows this algorithm:
+      </p>
+        <ol>
+          <li>
+Follow the algorithm defined in [[[SD-JWT]]] for Verification of the SD-JWT.
```

Comment:
```suggestion
Follow the algorithm defined in [[[SD-JWT]]] [[SD-JWT]] for verification of the SD-JWT.
```

Reply:
What the heck does the syntax `[[[SD-JWT]]] [[SD-JWT]]` do?  I have no idea what the difference between triple brackets and double brackets is, and why we would want both!

Reply:
The triple brackets pulls in the specification's title, the double brackets puts there a reference to the reference section (and fills the reference section with the right data like authors, year, URL, etc).

Reply:
I am keeping the capitalization in the phrase 'Verification of the SD-JWT' as that is a reference to the section title.

Reply:
@brentzundel -- Oh, I see. The *external* section title. In that case, I think the whole reference should change, a la --
```suggestion
Follow the algorithm defined in Section
<a data-cite="SD-JWT#name-verification-of-the-sd-jwt">Verification of the SD-JWT</a>
[[SD-JWT]].
```
`Section` might be a reasonable omission.